### ISSUE TITLE
htop crash when attempting to change meter display with F2

### ISSUE THREAD

**Issue by prijatelj** [hishamhm/htop#693]
I run htop 2.0.2 on Arch Linux, and I have recently been encountering problems with the meter display. It appears that the htop display does not overwrite all characters in a column, thus causing character artificats. For example, I am sometimes missing the end bracket at one of the cpu meters, or worse, the content in the right column, default stuff, is sometimes displayed inside the cpu meters, and the columns are no longer clearly defined. So the tasks row in the right column is partially in the right column cpu row and partially outside of it running off into what would be the right column.

As requested by the  htop output upon crash, here is the back trace:
```
htop(CRT_handleSIGSEGV+0x36)[0xea50fbc9986]
/usr/lib/libc.so.6(+0x34920)[0x658ddb07f920]
htop(String_freeArray+0x9)[0xea50fbc15c9]
htop(Header_writeBackToSettings+0x4f)[0xea50fbb9def]
htop(+0x19a7b)[0xea50fbc4a7b]
htop(+0xde5a)[0xea50fbb8e5a]
htop(ScreenManager_run+0x279)[0xea50fbbfb69]
htop(main+0x414)[0xea50fbb6604]
/usr/lib/libc.so.6(__libc_start_main+0xea)[0x658ddb06bf6a]
htop(_start+0x2a)[0xea50fbb683a]

```
And the htop.objdump, I am uncertain where to share this? I only found the bug tracker and the link they sent me to did not have an obvious submit bug. The htop.objdump is massive, so if there is no where else I'll put it here, but please follow up with me. I would love it if my htop would not get buggy with its displays, and would definitely love it if it would not crash. I used htop for a long time without these issues, I'd like to return to those good ol' days. :]

**Comment by hishamhm** [comment]
Hi, you can email me the htop.objdump file to h@hisham.hm — thanks!

**Comment by prijatelj** [comment]
I'm incredibly sorry, I deleted it after I deleted the htop user settings and uninstalled/reinstalled htop. htop doesn't crash now. I must have somehow messed up the user settings just by using htop and trying to move things around. I'm sorry I do not have it. If it occurs again, I will provide you with what you need.

Although now Htop does not crash, I still experience the overwriting problem in the display and do not know how to fix it. So the stuff in the right column, overlaps what is in the left column, without much rhyme or reason apparent by looking at it as it is. Anything else I could provide to help with this problem?

**Comment by hishamhm** [comment]
Fixed in 2.1.0! Thanks for reporting!
